sdcm/cluster.py

```diff
@@ -5113,6 +5114,17 @@ def decommission(self, node: BaseNode, timeout: int | float = None) -> DataCente
             return None
         return dc_topology_rf_change if tablets_enabled else None
 
+    def dc_remove(self, node: BaseNode, keyspaces, new_dc_name, timeout: int | float = None):
```

Comment:
`dc_remove` name is very generic while it requires very detailed arguments like node and dc name. `dc_remove` name would rather imply only datacenter name as argument only.

let's keep it as close as possible to documentation. E.g. should do `cluster_node.run_nodetool(sub_cmd="repair -pr", publish_event=True)`
Also, add docstring what are keyspaces and why needed (or maybe we could make it automatic - detect which keyspaces are using new dc)

Another thing to consider is to support multiple nodes